### DISCUSSION TITLE
How to remove the right side TOC nativation bar?

### DISCUSSION THREAD

**Discussion by wcj365** [executablebooks/meta#559] [Q&A]
My book does not use the right side navigation bar for the lower level table of contents. 
However, the white space is still reserved for it. 
I wonder if there is a way to claim the white space for my contents themself. 
Thanks!

**Comment by lwasser** [comment]
hey @chrisjsewell @choldgraf  I actually have this same question. is it possible to remove that right side contents TOC for the page? i'm just finding it busy visually.  (loving book theme however!)

**Comment by NickleDave** [comment]
This is a theme specific question, right?  
I don't find anything in the sphinx-book-theme docs on removing the right-side TOC.

Assuming the theme acts like `furo`, you maybe can suppress it on a page-by-page basis with a `hide-toc` directive?
https://pradyunsg.me/furo/customisation/toc/

But I guess that you want to do it site-wide

**Reply by lwasser** [reply]
@NickleDave yes I want to do it site wide. I'm probably going to end up customizing the theme anyway so perhaps there is an easy-ish way to do that if it can't be done with the html settings? but i thought id ask first here in case i'm missing something in the docs!

**Comment by NickleDave** [comment]
:+1: I would guess it would be here (which I'm also guessing you looked at already ... just adding a link for purposes of discussing)  
https://sphinx-book-theme.readthedocs.io/en/stable/customize/sidebar-secondary.html

**Reply by lwasser** [reply]
Yes... i spent some time on that page today. I also looked into adding scripts to the `<head></head>` of the page. none of this seems to work. It just occurred to me that I might be able to turn it off with css display:none maybe. 

One of the issues is 

i used `"show_toc_level": 1,` and also level 2 but it still seems to show more than 2 levels. And for the lower level headings i like to be super descriptive with titles for scannability (is that a word?!) and SEO.

<img width="343" alt="Screen Shot 2022-10-26 at 12 11 19 PM" src="https://user-images.githubusercontent.com/7649194/198078914-86956404-3e45-48a8-90aa-4de2618e1932.png">

my toc for our packaging guide is crazy long. But i also just think there's too much on the page to have nav on both sides and i'm worried about usability and accessibility (and design too).

**Reply by lwasser** [reply]
i think you know what i'm working on :) readme files for dayz.

**Reply by choldgraf** [reply]
Hey all - quick answer here: I don't think this is currently possible, but we have recently implemented this in the PyData theme, which is the upstream theme for the book theme:

https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/page-toc.html#remove-the-table-of-contents

@AakashGfude has recently been working on updating the book theme to use the latest PyData theme, and so when that happens we should be able to just "inherit" this feature.

**Reply by lwasser** [reply]
Thanks @choldgraf !! that's super helpful to know. I was able to hide it with CSS but not remove it. i'll stop trying and will wait for that update which sounds great. 

[Also i found this ](https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/fonts.html) while digging around the pydata theme docs - i'm interested in adding some custom google fonts to match our website. but to do that i need to add a few links to the `<head></head>` of the html file (ie the layout.html file i think?) to load the fonts. Is that possible to do without modifying the theme / creating a subtheme? 

many thanks. i'm bombaring y'all with questions today but thoroughly enjoying all of these tools that you and chris and everyone have developed. they are incredible!
